### ISSUE TITLE
Testing

### ISSUE THREAD

**Issue by Beanow** [Avaq/momi#6]
To clear CI/CD and security audit requirements.

**Comment by Avaq** [comment]
Set up CI, just need more tests

**Comment by Avaq** [comment]
Do the [current tests](https://github.com/Avaq/momi/blob/master/test/index.test.js) suffice? Current coverage is 77%.

**Comment by Avaq** [comment]
The two underlying libraries are both fully covered.

**Comment by Avaq** [comment]
I'll take your silence as a yes. ;)

**Comment by Beanow** [comment]
Ah, yes :sweat_smile: service bootstrapping was the main use-case at the time, for which I've switched to hooks and booture. So currently not using momi.

Given an opportunity to use this stack for another project and a round of refactoring, I'd be interested to see if momi could be used for smaller request handlers. For request handlers now I used fat sequential handlers with service calls instead of middleware style composition. Dependency injection of those services being the inversion of control point. Especially tests got a lot of duplication from this and helper functions emerged to control boilerplate of common patterns.
